json-rpc/Cargo.toml

```diff
@@ -35,8 +35,14 @@ libra-types = { path = "../types", version = "0.1.0" }
 libra-temppath = { path = "../common/temppath", version = "0.1.0", optional = true }
 libra-workspace-hack = { path = "../common/workspace-hack", version = "0.1.0" }
 move-core-types = { path = "../language/move-core/types", version = "0.1.0" }
+move-vm-types = { path = "../language/move-vm/types", version = "0.1.0" }
 network = { path = "../network", version = "0.1.0" }
 storage-interface = { path = "../storage/storage-interface", version = "0.1.0" }
+compiled-stdlib = { path = "../language/stdlib/compiled",  version = "0.1.0" }
+vm-genesis = { path = "../language/tools/vm-genesis", version = "0.1.0" }
+executor = { path = "../execution/executor", version = "0.1.0" }
+executor-types = { path = "../execution/executor-types", version = "0.1.0" }
+scratchpad = { path = "../storage/scratchpad", version = "0.1.0" }
```

Comment:
all these should be "optional" deps, since it is only used in test, right?  but it is kind of concerning that we are importing so many stuff in, all these should probably be moved intoto libradb package in the future.

Reply:
we have to keep them for now, because the tests dir is under src, so build need them.
I will have next diff to move tests out of src, then update the dependencies.

Reply:
my bad, missing cfg(test) for genesis file, fixed now.